h/services/group_list.py

```diff
@@ -199,19 +209,6 @@ def world_group(self, authority):
             .one_or_none()
         )
 
-    def _readable_by_world_groups(self, user=None, authority=None):
-        """
-        Return all groups readable by world for the authority.
-        """
-
-        authority = self._authority(user, authority)
-        groups = (
-            self._session.query(models.Group)
-            .filter_by(authority=authority, readable_by=group.ReadableBy.world)
-            .all()
```

Comment:
Good grief, so this was loading every single world-readable group on every page render!

Reply:
Yeeeeah.